templates/htmx-tags-dropdown.html

```diff
@@ -0,0 +1,12 @@
+<!-- Requires #search_tags_input to exist -->
+<div id="tags_dropdown" class="dropdown-menu dropdown-blue z-20 min-w-full !border-gray-400"
+    style="display: none; opacity: 0; border: none; top: 100%; padding: 0px !important; margin: 0px; margin-top: 0.25rem;"
```

Comment:
I wonder if by this point isn't it better to make this into a class? Or use tailwind?

Reply:
Yes, I would prefer that too, also to make it easier in the future to have a more similar layout in other places.

Reply:
Apart from that, I think we are good to go!

Reply:
i agree, maybe in a separate issue since there are multiple dropdowns

Reply:
Fair enough! Can you open an issue so we don't forget?